src/main/java/io/micronaut/build/MicronautQualityChecksAggregatorPlugin.java

```diff
@@ -41,12 +41,22 @@ private void configureSonar(final Project rootProject) {
                         p.property("sonar.organization", "micronaut-projects");
                         p.property("sonar.host.url", "https://sonarcloud.io");
                         p.property("sonar.java.source", "8");
+                        p.property("sonar.verbose", "true");
 
-                        final String xmlReportPath = rootProject.getBuildDir().toPath()
+                        // Jacoco integration
+                        final String jacocoReportPath = rootProject.getBuildDir().toPath()
                             .resolve("reports/jacoco/" + COVERAGE_REPORT_TASK_NAME + "/" + COVERAGE_REPORT_TASK_NAME + ".xml")
                             .toFile().getAbsolutePath();
-                        p.property("sonar.coverage.jacoco.xmlReportPaths",
-                            xmlReportPath);
+                        p.property("sonar.coverage.jacoco.xmlReportPaths", jacocoReportPath);
```

Comment:
I'll defer that to you, since you know better than me what should be changed.

Note that they don't use GitHub issues, but a community forum: https://community.sonarsource.com/c/help/8